### PR TITLE
Document new CPU limits

### PR DESCRIPTION
**Pull request by fkorotkov**
This will take effect on Sunday May 31st at 10 AM EST.

### CODE REVIEW

**Comment by fkorotkov** [comment]
More details are here: https://medium.com/cirruslabs/new-cpu-based-limits-for-cirrus-ci-b0dcd82f4615

File: docs/faq.md [hunk 1]
```diff
@@ -24,20 +24,27 @@ publish_task:
 
 ## Are there any limits?
 
-Cirrus CI has following limitations on how many VMs or Containers a single user can run on community clusters for public repositories for free or for private personal repositories under the [$10/month plan](https://github.com/marketplace/cirrus-ci/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45OTI=#pricing-and-setup):
+Cirrus CI has following limitations on how many CPUs for different platforms a single user can run on community clusters
```
Comment (RDIL) [inline docs/faq.md:27]:
```suggestion
Cirrus CI has the following limitations on how many CPUs for different platforms a single user can run on community clusters
```

File: docs/faq.md [hunk 2]
```diff
@@ -24,20 +24,27 @@ publish_task:
...
+* 4.0 CPUs macOS VM (2 VMs with hyper-threading).
+
+There are no limits on how many VMs or Containers you can run in parallel if you bring [your own infrastructure](guide/supported-computing-services.md)
 or use [Compute Credits](pricing.md#compute-credits) for either private or public repositories.
 
 !!! note "No per repository limits"
-    Cirrus CI doesn't enforce any limits on repository or organization levels. All the limits are on per user basis.
-    For example, if you have 10 active contributors to a repository then you can end up with 130 tasks running in parallel 
-    for the repository.
+    Cirrus CI doesn't enforce any limits on repository or organization levels. All the limits are on per-user basis.
```
Comment (RDIL) [inline docs/faq.md:47]:
```suggestion
    Cirrus CI doesn't enforce any limits on repository or organization levels. All the limits are on a per-user basis.
```

**Comment by fkorotkov** [comment]
Thank you!

**Pull request merged**